templates/internet-of-things/appstore.html

```diff
@@ -11,7 +11,7 @@
   <div class="p-strip--suru-half-and-half">
     <div class="row u-equal-height">
       <div class="col-7 col-medium-4">
-        <h1>White-label app stores<br class="u-hide--small" />for IoT</h1>
+        <h1>White-label app stores <br class="u-hide--small" />for Linux and IoT</h1>
         <p>Build a platform ecosystem for connected devices to unlock new avenues for revenue generation. We provide the infrastructure for secure management and seamless software updates for your fleet of devices. Whether you need a branded app store for your IoT devices or private application repository on premise, we can accommodate your requirements.</p>
```

Comment:
It looks like this sentence is "Manufacturers can cut the cost of managing software updates on their devices and take advantage of new revenue opportunities by selling services via their own branded IoT app store – hosted by Canonical." In the copy doc. Are these changes correct?